main.js

```diff
@@ -1,66 +1,68 @@
-  const lib = localStorage.getItem('library');
-  return lib ? JSON.parse(lib) : [];
-};
-const render = (books) => {
-  let booksList = '';
-  books.forEach((book, i) => {
-    const { title, author } = book;
-    booksList += `
+  // Display books and save in Local Storage
+
+  showBooks() {
+    const bookTable = document.getElementById('table-body');
+    let table = '';
+    this.bookList.forEach((book, i) => {
+      table += `
     <tr>
       <th scope="row">${i + 1}</th>  
-      <td>${title}</td>
-      <td>${author}</td>
-      <td ><span onClick="removeBook(${i})" class="btn btn-danger btn-sm delete del py-2 px-3">Delete</span></td>
+      <td>${book.title}</td>
+      <td>${book.author}</td>
+      <td><span id="${book.id}" onClick="remove(${
+  book.id
+})" class="btn btn-danger btn-sm delete del py-2 px-3">Delete</span></td>
     </tr>`;
-  });
-  return booksList;
-};
-const show = () => {
-  document.getElementById('table-body').innerHTML = render(getBooks());
-};
-const save = (book) => {
-  localStorage.setItem('library', JSON.stringify(book));
-  show();
-};
-// add a book to library array
-const addBook = (book) => {
-  const { title, author } = book;
+    });
+    bookTable.innerHTML = table;
+  }
 
-  const books = getBooks();
-  const dup = books.filter(
-    // eslint-disable-next-line comma-dangle
-    (item) => item.title === title && item.author === author
-  );
-  if (dup.length === 0 && title.length > 0) {
-    books.push({
+  // add a book to library array
+  addBook(title, author) {
+    const book = {
+      id: this.bookList.length + 1,
       title,
       author,
-    });
+    };
+
+    this.bookList = [...this.bookList, book];
+    localStorage.setItem('library', JSON.stringify(this.bookList));
   }
-  return save(books);
-};
 
-// delete a book
+  // delete a book
+  removeBook(index) {
+    this.bookList = this.bookList.filter((b) => b.id !== index);
+    localStorage.setItem('library', JSON.stringify(this.bookList));
+    this.showBooks();
+  }
+}
+
+const bookList = new BookLists();
+const titleInput = document.querySelector('#title');
+const authorInput = document.querySelector('#author');
+const addInput = document.querySelector('#button');
+
+bookList.showBooks();
+
 // eslint-disable-next-line no-unused-vars
-const removeBook = (bk) => {
-  const books = getBooks();
-  books.splice(bk, 1);
-  save(books);
-};
-bookForm.addEventListener('submit', (e) => {
+function remove(index) {
+  //eslint-disable-line
+  bookList.removeBook(index);
+}
+
+addInput.addEventListener('click', (e) => {
   e.preventDefault();
-  const book = new Book();
-  addBook(book);
-  bookForm.reset();
-  show();
+  if (titleInput.value && authorInput.value) {
+    bookList.addBook(titleInput.value, authorInput.value);
+    bookList.showBooks();
+    titleInput.value = '';
+    authorInput.value = '';
+  }
 });
 // load books list when page loads
-window.onload = show();
\ No newline at end of file
+// window.onload = show();
```

Comment:
[Optional]
- [ ] You should remove the commented-out code blocks to make your code base more professional. The git keeps track of your changes, you can easily see the removed code blocks from your git history.